.changeset/eleven-islands-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/tsyringe': minor
```

Comment:
How about setting this `major`? If this is `minor`, the next version will be `0.2.0`. I think it's good to be `1.0.0`.

Reply:
I agree that it is stable enough to `1.0.0`. The PR is updated.